### ISSUE TITLE
Error during the Get-TargetResource of SPManagedMetaDataServiceApp resource if ContentTypeHubUrl is null

### ISSUE THREAD

**Issue by luigilink** [PowerShell/SharePointDsc#445]
**Details of the scenario you try and problem that is occurring:**
Error during the Get-TargetResource of SPManagedMetaDataServiceApp resource.

**The DSC configuration that is using the resource:**
```
        SPManagedMetaDataServiceApp APPLICATION_SpsSvcAppManagedMetadataServiceApp
        {  
            PsDscRunAsCredential = $FarmSetup
            Name                 =  'MMS'
            ProxyName            = 'MMS'
            ApplicationPool      = 'Services Application Pool'
            DatabaseServer       = 'SQLSPS'
            DatabaseName         = 'SP15MMS'
        }
```

**Version of the Operating System and PowerShell the DSC Target Node is running:**
Windows 2012 R2

**Version of the DSC module you're using:**
SharePointDsc 1.4.0.0

```
Get-DscConfiguration : PowerShell DSC resource MSFT_SPManagedMetaDataServiceApp  failed to execute Get-TargetResource
functionality with error message: You cannot call a method on a null-valued expression.
At line:1 char:1
+ Get-DscConfiguration
+ ~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (MSFT_DSCLocalConfigurationManager:root/Microsoft/...gurationManager)
   [Get-DscConfiguration], CimException
    + FullyQualifiedErrorId : ProviderOperationExecutionFailure,Get-DscConfiguration
```

**Comment by luigilink** [comment]
Fix suggested in PR #446

**Comment by BrianFarnhill** [comment]
Fix merged, closing this one. Thanks dude
